examples/postgres.py

```diff
@@ -20,41 +15,35 @@ def __str__(self):
         return str(self.id)
 
 
-CONNECTION_PARAMS = {
-    'host': 'localhost',
-    'port': '54325',
-    'user': 'tortoise',
-    'password': 'qwerty123',
-    'database': 'tortoise',
-}
-
-
 async def run():
-    client = AsyncpgDBClient(single_connection=True, **CONNECTION_PARAMS)
-    test_db_name = CONNECTION_PARAMS['database'] + '_test'
-    await client.create_connection()
-
-    try:
-        await client.execute_script('DROP DATABASE {}'.format(test_db_name))
-    except InvalidCatalogNameError:
-        pass
-    await client.execute_script(
-        'CREATE DATABASE {} OWNER {}'.format(test_db_name, CONNECTION_PARAMS["user"])
-    )
-    await client.close()
-    test_db_connection_params = copy(CONNECTION_PARAMS)
-    test_db_connection_params['database'] = test_db_name
-
-    client = AsyncpgDBClient(single_connection=True, **test_db_connection_params)
-    await client.create_connection()
-    Tortoise.init(client)
-    await generate_schema(client)
+    await Tortoise.init({
+        'connections': {
+            'default': {
+                'engine': 'tortoise.backends.asyncpg',
+                'credentials': {
+                    'host': 'localhost',
+                    'port': '54325',
+                    'user': 'tortoise',
+                    'password': 'qwerty123',
+                    'database': 'test',
```

Comment:
Is it possible to use the db_url stuff I did?
It is a lot more concise, and since we changed over to DB-URLS at work it really made configuration much simpler, as e.g. with Docker you can easily inject environment variables, so we use that for configuration.

Reply:
I refactored `db_url` to `config_generator` - you can see how config generated for it in `test_db_url`

Reply:
I feel we should make the examples use that, as it is more concise.
Or just document both, a "shorthand" that is fine for most use cases, and an "advanced"?